drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -4131,9 +4131,6 @@ void RasterizerStorageGLES3::mesh_render_blend_shapes(Surface *s, const float *p
 	for (int ti = 0; ti < mtc; ti++) {
 		float weight = p_weights[ti];
 
-		if (weight < 0.001) //not bother with this one
```

Comment:
This is a terrible idea and it's going to kill performance

Reply:
@punto- Can you reply?

Reply:
Use: 
```cpp
		if (weight < 0.00001) //not bother with this one
			continue;
```